.github/workflows/windows.yml

```diff
@@ -37,7 +37,7 @@ jobs:
 
       - name: Display Changes
         if: failure()
-        run: git diff
+        run: git diff-files && git diff
```

Comment:
This is currently for testing and failure analysis. 'git diff' is not returning anything with the current problem so I added 'git diff-files' to see at least which files do have a problem and lead to failure in the preceding check. Unfortunately I cannot see what diffs there actually are in the files. This could be reverted once we resolve the problem but it might be helpful information in some cases.

Reply:
I see. I guess you could permanently replace the `--quiet` flag from the previous command with `--exit-code` for this: https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---quiet

Reply:
Changed to `run: git diff-files --exit-code`.